buildscripts/common.props

```diff
@@ -2,7 +2,6 @@
 
 	<PropertyGroup>
 		<NoWarn>$(NoWarn);CS1591;CS3014;CS3003;CS3001;CS3021</NoWarn>
-		<NoWarn>$(NoWarn);CS0612;CS0618</NoWarn> <!-- TODO: Remove this line once `[Obsolete]` members have been dealt with. -->
```

Comment:
Shouldn't we keep this so we can easily obsolete things in the future without having to re-add it back?

Reply:
Good thought. Let's leave this in for now, then.